style.css

```diff
@@ -1,3 +1,3 @@
 .text {
```

Comment:
- [ ] Just as I previously requested, kindly make an inline comment. This [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) would help you out. You can message me on slack for more clarification. I am Olamarx on Slack.

Reply:
ok what about now